tox.ini

```diff
@@ -19,9 +18,11 @@ setenv =
 deps =
     -rrequirements.txt
     -rtest-requirements.txt
-    ansible24: ansible==2.4.6.0
-    ansible25: ansible==2.5.9
-    ansible26: ansible==2.6.4
+    ansible24: ansible>=2.4,<2.5
+    ansible25: ansible>=2.5,<2.6
+    ansible26: ansible>=2.6,<2.7
+    ansible27: ansible>=2.7,<2.8
+    ansibledevel: git+https://github.com/ansible/ansible.git
 commands =
     unit: pytest test/unit/ --cov={toxinidir}/molecule/ --no-cov-on-fail {posargs}
```

Comment:
@gundalow just a note: I'd like to also drop `--no-cov-on-fail`, separately when integrating codecov...